scripts/postNpmInstall.js

```diff
@@ -0,0 +1,10 @@
+// Remove the webpack-dev-server grunt task since we do not use it
+var fs = require('fs');
+var path = require('path');
+
+var taskFile = path.join(__dirname, '..', 'node_modules', 'grunt-webpack', 'tasks', 'webpack-dev-server.js');
```

Comment:
Tried a bit more, and `require.resolve('grunt-webpack/tasks/webpack-dev-server.js')` *does* work. I'll just have to either try/catch around it or pass some option to turn off exception throwing if the file isn't found. 👍